en/mkdocs.yml

```
@@ -258,6 +258,14 @@ nav:
                 - HTTP 2.0: deploy-and-publish/deploy-on-gateway/choreo-connect/http-2.0.md
                 - Rate Limiting:
                     - Enabling distributed throttling: deploy-and-publish/deploy-on-gateway/choreo-connect/rate-limiting/distributed-throttling.md
+                - Message Transformation:
+                    - Message Transformation: deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/message-transformation-overview.md
+                    - Interceptor Microservice:
+                        - Interceptor Microservice: deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/interceptor-microservice/interceptor-microservice.md
+                        - Request Flow Interceptor: deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/interceptor-microservice/request-flow-interceptor.md
+                        - Response Flow Interceptor: deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/interceptor-microservice/response-flow-interceptor.md
+                        - Sample Interceptor Services: deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/interceptor-microservice/sample-interceptor-services.md
```

```suggestion
                        - Sample Interceptor Services: deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/interceptor-microservice/sample-interceptor-services.md
                        - Sample OpenAPI Definition for the Interceptor: deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/interceptor-microservice/sample-interceptor-services.md
```
After the initial introduction, it would be helpful to have direct access to this when actually writing the interceptor service.

---

This page should be deleted